tests/chaos/conftest.py

```diff
@@ -377,8 +377,11 @@ def chaos_vms_instancetype_list(request, admin_client, chaos_namespace):
             image=Images.Rhel.RHEL9_REGISTRY_GUEST_IMG,
             vm_instance_type=required_instancetype,
         )
+        vm.deploy(wait=True)
```

Comment:
You can add this in [test_deploy_vm](https://github.com/RedHatQE/openshift-virtualization-tests/blob/7287c85d351d6db3b1fa4ffd4c781eb0377e95cc/tests/chaos/standard/test_standard.py#L163)
This way you only need to change `wait=False` -> `wait=True`
And it should have the same effect